### ISSUE TITLE
After resize screen 400% only horizontal scroll bar is appearing.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1024]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/528409

**Comment by ato9000** [comment]
URL:https://ms.portal.azure.com/

User Experience: 
If user navigate on page and after resize screen 400% and controls and focus does not visible properly, so it would difficult for user to navigate on page.

Pre Requisite:-
Go to browser setting and zoom screen 400%

Repro Steps:
1. Open URL: https://ms.portal.azure.com/

2. Login via valid credentials
3. Navigate to "Azure Cosmos DB " from left pane and select it.
4. Navigate to any of the Azure Cosmos DB which is already created and select it.
5. Navigate to "Data Explorer" from the left and select it.
6. Navigate  to Settings menu item and select it under Data explorer blade.
7. Navigate to all the control after zoom in to 400% and verify the issue.

Actual Result: 
After resize screen 400% controls only horizontal scroll bar is appearing instead of vertical scroll bar.

Expected Result:
After resize screen 400% controls should align properly and only vertical scroll bar should appear and user should able to see and access all the controls on the screen.

Note:Same issue exist throughout the application

MAS References:
MAS 1.4.10 - Reflow ***

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.


External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link.


https://user-images.githubusercontent.com/8519407/132721274-3ea02655-82ac-4198-ab05-d7c68aff77e8.mp4
